mne/chpi.py

```diff
@@ -1383,3 +1384,40 @@ def _compute_good_distances(hpi_coil_dists, new_pos, dist_limit=0.005):
             exclude_coils = np.where(use_mask)[0][np.argmax(badness)]
             use_mask[exclude_coils] = False
     return use_mask, these_dists
+
+
+def get_active_chpi(raw):
+    """Determine how many HPI coils were active for a time point.
+
+    Parameters
+    ----------
+    raw : instance of Raw
+        Raw data
+
+            .. versionadded:: 1.2
+
+    Returns
+    -------
+    times : array, shape (n_times)
+        The number of active cHPIs for every timepoint in raw.
+    """
+    # get meg system
+    system, _ = _get_meg_system(raw.info)
+
+    # processing neuromag files
+    if system in ['122m', '306m']:
```

Comment:
A better pattern is:
```
if system not in ...:
    raise
chpi_info = ...
```
then the rest of the code follows. That way you save an indentation level